requirements.txt

```diff
@@ -1,5 +1,6 @@
 Flask==0.12.2
 Flask-JWT==0.3.2
+git+https://github.com/atengler/flask-swagger-ui.git@3.4.3-update#flask_swagger_ui
```

Comment:
Fuj, is this really necessary? We need to have this in `setup.py` as well.

Reply:
We can get rid of this after this is merged, as we discussed yesterday - https://github.com/sveint/flask-swagger-ui/pull/14